transform/transformers/pck_transformer.py

```diff
@@ -142,6 +142,19 @@ def round_currency_values(self):
                 for k, v in self.data.items()  # noqa
                     if k in self.rsi_currency_questions or k in self.qcas_currency_questions})  # noqa
 
+    def parse_negative_values(self):
+        """If any number field contains a negative value then replace it with a number containing
+        only 9's of the same length (i.e., -85 = 99, -1500 = 9999, -934000 = 999999)
+        """
+        for k, v in self.data.items():  # noqa
+            try:
+                # If v isn't a number then an exception is thrown and we skip it
+                int_v = int(v)
+                if int_v < 0:
+                    self.data[k] = '9' * len(str(abs(int_v)))
```

Comment:
I'm pretty sure the requirement is actually to replace negative values with the maximum number of nines that downstream supports. In other words any negative value is replaced with 9999999999.
(i.e., -85 = 9999999999, -1500 = 9999999999, -934000 = 9999999999)

Reply:
🤦‍♂️ Did you find that written down somewhere?  I'm fairly sure I asked those questions but I'll double check.  Also it's super easy to change so it's not a big deal ^^

Reply:
This is kind of my point, what are you working from? My knowledge is just from discussions held during the meeting where this requirement was discussed. Also from very old knowledge of how the paper system worked when I supported that many moons ago.